shared-files/_glossary.md

```diff
@@ -0,0 +1,324 @@
+            </li>
+        </ol>
+    </dd>
+    <p>
+        <b>Synonyms:</b> <i>Apps, Apps & Marketplace</i>
+    </p>
+    <dt>
+        Cloud credentials
+    </dt>
+    <dd>
+        Placeholder
+    </dd>
+    <p>
+        <b>Related terms:</b> <i>Hosted clusters</i>
+    </p>
+    <dt>
+        Community
+    </dt>
+    <dd>
+        A build of Rancher that's available to the entire Rancher community.
+    </dd>
+    <p>
+        <b>Related terms:</b> <i>Prime, Rancher Prime</i>
+    </p>
+</dl>
+
+## D
+
+<dl>
+    <dt>
+        Downstream cluster
+    </dt>
+    <dd>
+        An alternative name for a <i>Managed</i> cluster: a Kubernetes cluster that is managed by a Rancher server.
+    </dd>
+    <p>
+        <b>Synonyms:</b> <i>Managed cluster</i>
+    </p>
+    <p>
+        <b>Related terms:</b> <i>Hosted cluster, Imported cluster, Rancher server, Registered cluster</i>
+    </p>
+</dl>
+
+## E
+
+<dl>
+    <dt>
+        Extension
+    </dt>
+    <p>
+        <b>Versions:</b> Rancher v2.7.0 and later
+    </p>
+    <dd>
+        Helm charts that can extend and enhance the Rancher UI, and tailor Rancher to your specific environment. Some Extensions require manual installation on a cluster, while others are built into Rancher. Built-in Extensions include Fleet and Harvester. Manually installed Extensions inlcude Kubewarden and Elemental.  
+    </dd>
+    <p>
+        <b>Related terms:</b> <i>Apps & Marketplace, Catalogs, Integrations</i>
+    </p>
+</dl>
+
+## H
+
+<dl>
+    <dt>
+        Hosted cluster
+    </dt>
+    <dd>
+        A managed cluster that is hosted in the cloud, by a Kubernetes provider. Rancher supports a variety of hosted Kubernetes providers, including Amazon Elastic Kubernetes Service (EKS), Azure Kubernetes Service (AKS), and Google Kubernetes Engine (GKE). Rancher integrates with the hosted Kubernetes provider's cloud APIs, so that you can manage the cluster from the Rancher UI.
+    </dd>
+    <p>
+        <b>Related terms:</b> <i>Managed cluster</i>
+    </p>
+</dl>
+
+## I
+
+<dl>
+    <dt>
+        Imported cluster
+    </dt>
+    <p>
+        <b>Versions:</b> Rancher 2.0–v2.4
+    </p>
+    <dd>
+        An existing cluster that was provisioned by a third-party, then set up to be managed by Rancher. <i>Registered clusters</i> in later versions of Rancher are roughly synonymous with imported clusters. The main difference is that there are more management features exposed in the Rancher UI for registered clusters than for imported clusters. 
+    </dd>
+    <p>
+        <b>Synonyms:</b> Registered cluster
+    </p>
+    <p>
+        <b>Related terms:</b> <i>Managed cluster</i>
+    </p>
+</dl>
+
+## K
+
+<dl>
+    <dt>
+        K3s
+    </dt>
+    <dd>
+        A lightweight Kubernetes distribution chiefly designed for workloads that run unattended under tight resource constraints.
+    </dd>
+    <p>
+        <b>Related terms:</b> <i>RKE</i>
```

Comment:
```suggestion
        <b>Related terms:</b> <i>RKE2</i>
```

If we're only listing one, RKE2 has a much closer association with K3s than RKE1.